packages/inspection-report/src/components/common/Thumbnail.js

```diff
@@ -0,0 +1,81 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Image, Text, StyleSheet, TouchableOpacity, View, useWindowDimensions } from 'react-native';
+import { MaterialIcons } from '@expo/vector-icons';
+import { useTranslation } from 'react-i18next';
+
+const styles = StyleSheet.create({
+  noImageWrapper: {
+    alignItems: 'center',
+    backgroundColor: '#f5f5f5',
+    borderRadius: 4,
+    justifyContent: 'center',
+  },
+  text: {
+    color: '#fff',
+    fontSize: 14,
+    marginTop: 5,
+    textAlign: 'center',
+  },
+});
+
+const responsiveImageSizes = [
+  { breakpoint: 480, width: 109 },
+  { breakpoint: 768, width: 140 },
+  { breakpoint: null, width: 109 },
+];
+
+function Thumbnail({ image, click }) {
+  const { t } = useTranslation();
+  const { width } = useWindowDimensions();
+
+  const imageSize = useMemo(
+    () => responsiveImageSizes.find((r) => (!r.breakpoint || width <= r.breakpoint)).width,
+    [width],
+  );
+
+  return (
+    <TouchableOpacity onPress={() => click(image)}>
+      <View style={{ width: imageSize }}>
+        {
+          !image.url ? (
+            <View style={[{ width: imageSize, height: imageSize }, styles.noImageWrapper]}>
+              <MaterialIcons
+                name="add-a-photo"
+                size={imageSize / 3}
+                color="#757575"
+              />
+            </View>
+          ) : (
+            <Image
+              source={{
+                width: imageSize,
+                height: imageSize,
+                uri: image.url,
+              }}
+            />
+          )
+        }
+        <Text style={styles.text}>{t(`gallery.parts.${image.label}`)}</Text>
```

Comment:
No, if you take a look at the specifications in the Jira ticket, the translations are contained in the image given as a prop. The pictures look like this :

```typescript
interface ExamplePicture {
  height: number;
  width: number;
  url: string;
  mimetype: string;
  label?: {
    fr: string;
    en: string;
  };
}
```

So you should have something like :

```javascript
function Thumbnail({ image, click }) {
  const { i18n } = useTranslation();
  const label = useMemo(() => (image.label ? image.label[i18n.language] : ''), [i18n]);

  ...
    <Text>{label}</Text>
  ...
}
```

You can also remove all the translations of the car parts in the resources folder

Reply:
Done